ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/InMemory.hs

```diff
@@ -148,32 +149,33 @@ data LedgerDB (l :: LedgerStateKind) = LedgerDB {
       -- | Ledger states
       ledgerDbCheckpoints :: AnchoredSeq
                                (WithOrigin SlotNo)
-                               (Checkpoint l)
-                               (Checkpoint l)
+                               (Checkpoint (l ValuesMK))
+                               (Checkpoint (l ValuesMK))
     , ledgerDbChangelog   :: DbChangelog l
+    , runDual             :: Bool
```

Comment:
This is a flag that will control whether to run both implementations or only the new one. In principle this is just a temporary workaround to avoid adding a new flag to the node, etc as it is just for development period.